data/formats-data.js

```diff
@@ -2183,6 +2183,16 @@ exports.BattleFormatsData = {
 		],
 		tier: "New",
 	},
+	smearglepokestar: {
```

Comment:
Yeah, that's pretty unfortunate. I'll note that it's a cosmetic forme only visible in Pokéstar in the dex one day, but having it this way in the code is better.